packages/runtime/src/widgets/Text.tsx

```diff
@@ -11,9 +11,6 @@ export type TextProps = {
 
 export const Text: React.FC<TextProps> = (props) => {
   const [text, setText] = useState(props.text);
-  if (props.text !== text) {
-    setText(props.text);
-  }
```

Comment:
Not sure why you're removing this?

Reply:
Peter Parker text doesn't change to Spiderman in presence of this condition

Reply:
Keep it as it was before for now, I will update this

Reply:
alright